src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -244,7 +244,7 @@
     - name: Deploy source code to a new deployment of an app. This will pack current directory, build binary with Pivotal Build Service and then deploy.
       text: az spring-cloud app deployment create -n green-deployment --app MyApp -s MyCluster -g MyResourceGroup
     - name: Deploy a pre-built jar to an app with jvm options and environment variables.
-      text: az spring-cloud app deployment create -n green-deployment --app MyApp -s MyCluster -g MyResourceGroup --jar-path app.jar --jvm-options="-XX:+UseG1GC -XX:+UseStringDeduplication" --env foo=bar
+      text: az spring-cloud app deployment create -n green-deployment --app MyApp -s MyCluster -g MyResourceGroup --artifact-path app.jar --jvm-options="-XX:+UseG1GC -XX:+UseStringDeduplication" --env foo=bar

```

Comment:
if there test covering this param? if yes, pls re-record test also.

Reply:
The test will still use the old/new arguments side by side to keep the backward compatibility. In next minor release, we will remove the old argument in the test case to completely.